### DISCUSSION TITLE
Can't core Update due to 401 Error

### DISCUSSION THREAD

**Discussion by jamfx** [akeeba/panopticon#347] [Q&A (Community Support)]
Hi, 
Something went wrong on a Site I watch with Panopticon: The Core-Update fails with the following Erros:
Client error: `POST https:// my domain . tld / administrator/components/com_joomlaupdate/extract.php` resulted in a `401 Unauthorized` response: Unauthorized 

Updates for Extensions working fine. AdminTools is working. Exeption for the extract.php is included in .htaccess

What did I do wrong?
best regards
Niko
PS: Updating from within Joomla Backend works fine!

**Comment by nikosdion** [comment]
The HTTP 401 means that your server does not recognise the .htaccess username or password.

Hm. Try trimming your password to only include the first 8 characters. Yes, I know it sounds weird. I've seen some Apache servers which, for whatever reason, are stuck using the old `crypt` library which only hashes the first 8 characters of a password.

**Comment by jamfx** [comment]
Hi Nicholas, 
that sounds really wired :grin: And I don't really think, this will work. I will explain why: 
This issue is only on one Site of many other Sites on my (the same) Machine. All other updates on similar configurations (made by me) went smooth. And I found out: Same happens on a Site (on another [not my] Server) from 4.4.0 to 4.4.1. 
So I would point out something else. Maybe an .htaccess-Issue (like 401 seems to point it out clearly). But I don't understand why. Updates from Extensions on the same Site: working! Core-Update was not...

**Reply by nikosdion** [reply]
I can tell you it's not Panopticon, the Joomla! version, or the admin password. While I was having dinner Panopticon on my dev server did its business updating my sites from 4.4.0 to 4.4.1 — I didn't even realise there was a new Joomla! version LOL!

One of the sites is on Rochen, no admin password. One is on Rochen _with_ an admin password set up by Admin Tools. Two are on my local server, one with an admin password set by Admin Tools, and the other without. All four updated just fine.

**Comment by jamfx** [comment]
I am with you, as the same happens on my Sites from the same Machine. I thought you might have a clue, where to doublecheck and find the issue. As most updates went through smoothly but a very few failed with the above mentioned Error. And all my sites have a .htaccess-protected Backend :wink:

**Reply by nikosdion** [reply]
Did you try to reschedule the update, just in case the server burped?

Is it possible that the failing sites have passwords with characters other than a-z, 0-9, underscores, and dashes but the ones which worked only had those characters in them?

**Reply by jamfx** [reply]
I tried a few times to "unburb the Server" and rescheduled. But failed again.
And the passwords are like "MyS3cr3+Pa$w0rD_something-Site-Special" The first Characters are same on some more sites but the part after the underscore is more customer-spezific Text

**Comment by nikosdion** [comment]
This seems to be a .htaccess issue, already addressed with the information in https://github.com/akeeba/panopticon/discussions/348
